.github/workflows/pipeline-deploy-app-develop.yaml

```diff
@@ -38,31 +38,54 @@ jobs:
         action: apply
       secrets: inherit
 
-  build-and-deploy-react-app-to-s3-bucket:
-    name: Build and deploy react app to S3 bucket
+  build-react-app-to-s3-bucket:
+    name: Build react app to S3 bucket
     needs: [
       metadata,
       deploy-application-infrastructure,
     ]
-    uses: NHSDigital/uec-dos-management/.github/workflows/build-and-deploy-react-app.yaml@v0.18
+    uses: NHSDigital/uec-dos-management/.github/workflows/build-react-app.yaml@v0.23
     with:
       environment: dev
       workspace: ${{ needs.metadata.outputs.workspace }}
       domain: uec-cm
+      commit_hash: ${{ needs.metadata.outputs.commit_hash }}
+      artefact_bucket_name: ${{ needs.metadata.outputs.artefact_bucket_name }}
     permissions:
       id-token: write
       contents: read
     secrets: inherit
+
+  deploy-react-app-to-s3-bucket:
```

Comment:
The job to push (to artefact bucket) is build-react-app-tos3-bucket ; the job to deploy (to a different s3 SPA bucket) is deploy-react-app-to-s3-bucket. This is maybe a bit confusing around which bucket and the purpose of each bucket - maybe it could be build-and-push-react-artefacts and deploy-react-app  - with descriptions to match. Just a thought. If these do change then the slack notiication job will need to reflect the new names

Reply:
Job names renamed as the following: build-react-app-to-s3-bucket --> build-and-push-react-artefacts & deploy-react-app-to-s3-bucket --> deploy-react-app